app/src/main/java/org/jellyfin/mobile/webapp/RemotePlayerService.kt

```diff
@@ -263,6 +264,11 @@ class RemotePlayerService : Service(), CoroutineScope {
                 } else {
                     setPriority(Notification.PRIORITY_LOW)
                 }
+
+                if (AndroidVersion.isAtLeastS) {
+                    setForegroundServiceBehavior(Notification.FOREGROUND_SERVICE_IMMEDIATE)
+                }
```

Comment:
Might not be necessary [since the `foregroundServiceType` in the manifest is `mediaPlayback`](https://developer.android.com/develop/background-work/services/foreground-services#notification-immediate).

Reply:
Oh yeah thank you, that's indeed redundant. Removed that and fixed the lint issues.